src/Camera_TEST.cc

```diff
@@ -178,6 +178,17 @@ void CameraTest::RenderTexture(const std::string &_renderEngine)
   camera->SetImageHeight(80u);
   EXPECT_EQ(80u, camera->ImageHeight());
 
+  double height = 80;
+  camera->SetImageHeight(height);
+  double aspectRatio = camera->ImageWidth() / height;
+  EXPECT_NEAR(aspectRatio, camera->AspectRatio(), 1e-6);
```

Comment:
Since your new tests are almost the same as the previous test, lets remove the new test and continue on the previous one. E.g.,
```suggestion
  unsigned int height = 80;
  camera->SetImageHeight(height);
  EXPECT_EQ(height, camera->ImageHeight());
  double aspectRatio =
    static_cast<double>(camera->ImageWidth()) / static_cast<double>(height));
  EXPECT_NEAR(aspectRatio, camera->AspectRatio(), 1e-6);
```

The same should be done for width.

Also, go ahead and update this expectation below to use the new variables: https://github.com/gazebosim/gz-rendering/blob/8d4e5425f86b6ff4885eb8362fc74c1ee21d993d/src/Camera_TEST.cc#L195

e.g., `EXPECT_EQ(width*height*3u, ...)`